packages/fxa-settings/src/pages/ResetPasswordConfirmed/index.tsx

```diff
@@ -0,0 +1,42 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from 'react';
+import { RouteComponentProps } from '@reach/router';
+import { Localized } from '@fluent/react';
+import { usePageViewEvent } from '../../lib/metrics';
+
+type ResetPasswordConfirmedProps = {
+  serviceName: String;
+};
+
+// eslint-disable-next-line no-empty-pattern
+const ResetPasswordConfirmed = ({
+  serviceName,
+}: ResetPasswordConfirmedProps & RouteComponentProps) => {
+  usePageViewEvent('pw-reset-confirmed.confirmation');
+
+  // I feel like we need at least one of the Ready view variations in here, possibly one enabling action?
```

Comment:
I am not sure about these comments. Do you want to leave them, remove them, file a follow up, or maybe still want to address this in the PR?

Reply:
Oh this is still a WIP, I'll be taking them out as I resolve the issues! Sorry about that!